game/scripts/vscripts/modifiers/modifier_out_of_duel.lua

```diff
@@ -0,0 +1,21 @@
+modifier_out_of_duel = class({})
+
+function modifier_out_of_duel:CheckState()
+  return {
+    [MODIFIER_STATE_OUT_OF_GAME] = true,
+    [MODIFIER_STATE_COMMAND_RESTRICTED] = true,
+    [MODIFIER_STATE_ATTACK_IMMUNE] = true
+  }
+end
+
+function modifier_out_of_duel:IsHidden()
+  return false
```

Comment:
Hey, this isn't hidden and it doesn't have a tooltip. You cheated the Travis test. :O